test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/ScalaJSDefinedTest.scala

```diff
@@ -565,6 +565,32 @@ class ScalaJSDefinedTest {
     assertEquals(100, dyn.foo())
   }
 
+  @Test def properties_are_configurable(): Unit = {
+    // Named classes
+    @ScalaJSDefined
+    class Foo extends js.Object {
+      var myProp: Int = 1
```

Comment:
This creates an actual *field*, not a property. You should have a `def` without `()` instead to create an actual getter.